### PR TITLE
Update to allow usage of dflydev/fig-cookies v3

### PR DESCRIPTION
**Pull request by farums**
|    Q          |   A
|-------------- | ------
| Documentation | yes/no
| Bugfix        | yes/no
| BC Break      | yes/no
| New Feature   | yes/no
| RFC           | yes/no
| QA            | yes/no

### CODE REVIEW

**Comment by farums** [comment]
"mezzio/mezzio-session": "^1.4"
-->        "dflydev/fig-cookies": "^2.0.1",

**Comment by BusterNeece** [comment]
Would love to get this merged as it's one of the last few blockers to us migrating smoothly to PHP 8.

**Comment by samsonasik** [comment]
The PR title seems need to be updated to something like: "Allow dflydev/fig-cookies 3.0.0"

File: composer.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     },
     "require": {
         "php": "^7.3 || ~8.0.0",
-        "dflydev/fig-cookies": "^2.0.1",
+        "dflydev/fig-cookies": "^2.0.1 || ~3.0.0",
```
Comment (samsonasik) [inline composer.json]: I think it can uses : 

```suggestion
        "dflydev/fig-cookies": "^2.0.1 || ^3.0.0",
```

**Comment by weierophinney** [comment]
Closing in favor #14 (to allow GHA to run)

**Pull request closed without merging**